docker/build_standalone-tcks.sh

```diff
@@ -80,7 +80,7 @@ for tck in ${TCK_LIST[@]}; do
     DOC_SPECIFIC_PROPS="-propertyfile $BASEDIR/install/jaxrpc/bin/build.properties -Dts.home=$BASEDIR -Ddeliverable.version=1.1 -Ddeliverable.class=com.sun.ts.lib.deliverable.jaxrpc.JAXRPCDeliverable"
   elif [ "jta" == "$tck" ]
   then
-    TCK_SPECIFIC_PROPS="-Djta.classes=$GF_HOME/glassfish5/glassfish/modules/javax.transaction-api.jar;$GF_HOME/glassfish5/glassfish/modules/javax.servlet-api.jar"
+    TCK_SPECIFIC_PROPS="-Djta.classes=$GF_HOME/glassfish5/glassfish/modules/jakarta.transaction-api.jar;$GF_HOME/glassfish5/glassfish/modules/javax.servlet-api.jar"
```

Comment:
Is this jar name change (from javax.transaction-api.jar to jakarta.transaction-api.jar) in GlassFish expected?

Reply:
I'm not sure myself, as [New-Maven-Coordinates](https://github.com/eclipse-ee4j/ee4j/wiki/New-Maven-Coordinates) talks about changing the maven coordinates but not actual jar names.  I like the new jar name introduced by this PR but I'm curious if the GlassFish project plans to do the jar rename for all (Jakarta) spec api jars?

Reply:
The Jar that goes into maven central is just the name of the artifact in a similar way to normal for Maven projects. @anajosep what do you think to this PR and the need to change the Jar name? /cc @m0mus